mmv1/products/artifactregistry/VPCSCConfig.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'VPCSCConfig'
+description: |-
+  The Artifact Registry VPC SC config that applies to a Project.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/artifact-registry/docs/reference/rest/v1/VPCSCConfig'
+min_version: beta
```

Comment:
Is this resource intended to be set as Beta? From looking at the [Artifact Registry API docs](https://cloud.google.com/artifact-registry/docs/reference/rest/v1/projects.locations/updateVpcscConfig) I can only see a `v1` version of the API listed in the nav in on the left of the page, and no beta version.

Unless I'm missing something, this means we should remove `min_version: beta` here, and on L63 so that this resource is added to both the GA and Beta versions of the Google provider. This will also make the automated testing on the PR more straightforward 🎉

Reply:
This is intended. We are using google-beta for features in a public preview phase.